### PR TITLE
Add RPM specfile language

### PR DESCRIPTION
**Pull request by evitalis**
I was not 100% sure where to add the code to test said file but am happy to add it to this PR after some guidance if desired.

I work pretty regularly with RPMs and figured this would be helpful for myself and others.

### CODE REVIEW

**Comment by XAMPPRocky** [comment]
@evitalis Thank you for your PR! You don't need to add any code you just a comment on the top of the file that says what stats the file should have. You can look at the other files in `tests/data` such as the [rust one](https://github.com/XAMPPRocky/tokei/blob/master/tests/data/rust.rs#L1) for an example.

Also if you could, could you change `"line_comment: ["#"]` to `"base": "hash"`? it does functionally the same thing but reduces the amount of code generated at compile time.

**Comment by evitalis** [comment]
@XAMPPRocky I went ahead and added the counts. I also squashed my commits to make it a bit cleaner in the history.

**Comment by XAMPPRocky** [comment]
@evitalis There's a small error in your JSON, `"base"` is expected to be a single string and not an array. :)

**Comment by evitalis** [comment]
@XAMPPRocky Sorry for the back and forth hopefully this is in order now. I added a commit fixing the base definition.

File: tests/data/rpmspec.spec [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# 43 lines 22 code 4 comments 16 blanks
```
Comment (XAMPPRocky) [inline tests/data/rpmspec.spec]: This should be 42 lines.

Comment (evitalis) [reply tests/data/rpmspec.spec]: Fixed

**Comment by XAMPPRocky** [comment]
Thank you again for your PR and congrats on your first contribution! 🎉

**Pull request merged**